epub33/core/index.html

```diff
@@ -1213,6 +1213,29 @@ <h3>Exempt resources</h3>
 							CSS style sheets) is an exempt resource.</p>
 					</dd>
 
+					<div class="proposed correction" id="change_1">
+						<span class="marker">Proposed Correction 1</span>
+						<p>The exemption for resources not referenced from the spine or embedded in content documents
+							(refer to the paragraphs following note below), includes resources referenced from
+								<code>script</code> elements, as well as resources retrieved by scripts for processing.
+							The description focused on data sets travelling in the EPUB container, but that was only one
+							use case. This change adds scripting to the major exemption groups and removes it from the
+							case for resources travelling in the container. For more information, refer to <a
+								href="https://github.com/w3c/epub-specs/issues/2649">issue 2649</a>.</p>
+					</div>
+
+					<ins cite="#change_1">
+						<dt id="exempt-script">Scripts and script resources</dt>
+						<dd id="confreq-resources-cd-fallback-script">
+							<p>Any resource referenced from the [[html]] <a data-cite="html#the-script-element"
+										><code>script</code></a> element that is not already a core media type resource
+								(e.g., JavaScript) is an exempt resource.</p>
+							<p>In addition, any resources retrieved by a script (e.g., using an <code>import</code>
+								declaration [[ecmascript]] or the XmlHttpRequest [[xhr]] or Fetch [[fetch]] APIs) are
```

Comment:
I am not super happy with the examples here, since it seems to confuse exempt and foreign resources. At the very least it is a bit murky. Maybe just the import example? Or drop the examples since people seem to get very confused by them and decide it is an exhaustive list.

Reply:
I suppose a script could read a foreign resource that is embedded somewhere else in the publication, which would make it bypass the exemption.

It might be better to leave the second paragraph out and let the exemption below continue to handle exempt files for processing. I can't think of how we make this clearer without duplicating the two exemption scenarios.

Reply:
Rereading it this is probably fine as is. Do people really have to list remote resources loaded via XmlHttpRequest in the manifest? I guess they do. Scripting is horrible.

Reply:
> Do people really have to list remote resources loaded via XmlHttpRequest in the manifest? I guess they do.

Yup, that's my take, too. Used in the rendering = publication resource = manifest listing required.

> Scripting is horrible.

Ya, hoping we could get by in 3.0 underspecifying how scripting works wasn't the best of ideas.

What makes it more confusing is that you can't host most resources outside the container, unless, seemingly, you fetch them by script. So, if you want remote images, just fetch them and write them into a data url in an `img` tag. Spec rules circumvented.